lib/OpenQA/Utils.pm

```diff
@@ -558,5 +560,20 @@ sub job_groups_and_parents {
     return \@res;
 }
 
+sub wait_with_progress {
+    my $interval = ceil(shift);
+    my $tics;
+    $| = 1;
```

Comment:
It's better to just localize globals (`local $| = 1;`), then you reset to the original state and don't need to do it manually.